### ISSUE TITLE
Cannot change appearance of some AuiNotebook elements

### ISSUE THREAD

**Issue by mdcutone** [wxWidgets/Phoenix#1717]
**Operating system**: Windows
**wxPython version & source**: 4.1.0 from PyPI
**Python version & source**: 3.6

**Description of the problem**: 
AuiNotebook doesn't have a (simple) way to configure certain elements of its appearance (e.g. panel border and panel background). These elements are drawn by a nested AuiManager/AuiDockArt provider in the AuiNotebook class that are hidden away from the user. A workaround is to set the art provider directly after creating the notebook:

```python
notebook._mgr.SetArtProvider(myDockArt)
```
While this workaround does give the desired effect, it is undocumented and not evident without reading through the source code. Possibly extending the AuiTabArt provider with more options and having it configure the nested dock art provider automatically may be a better solution.

**Comment by Metallicow** [comment]
Yes as far as themeing, there are a few options that are not available to users. I patch these up in my applications by forcing the _private attributes. But yes at some point these other attriblutes that don't have a public doc'd way to get changed should be an item of a pull request. They been this way for some time, but I don't ever seem to get the time to rereview which ones still need added.

**Comment by mdcutone** [comment]
It would be really helpful to have additional setting exposed through the API. You can create very nice, modern looking UI themes with AUI once you dig around the source code and hack things. 

Been using wxPython for +10 years and I appreciate the work so far, particularly in the latest releases!

**Comment by infinity77** [comment]
Adding more setters/getters would not be a problem. That said, it would be nice to see an example of what you mean (visually). It could also be a nice way to contribute back to wxPython if you have some fancy tabarts/do Marta for AUI...

**Comment by mprosperi** [comment]
hello, when I wanted to change the background colour of toolbars and notebooks from old fashioned metal grey to white I had to change aui_utilities.GetBaseColour directly. It would be nice if it could be overloaded

Marco
